web_sdk/analysis_options.yaml

```diff
@@ -0,0 +1,5 @@
+include: ../analysis_options.yaml
+
+linter:
+  rules:
+    avoid_print: false
```

Comment:
This seems surprising. It might be better to enforce this to avoid leaking local development prints vs. intended logs.

Reply:
I removed this global opt-out and instead opted out the offenders on a file/line bases.